### ISSUE TITLE
Feature request: add an import explicitly.

### ISSUE THREAD

**Issue by alecthomas** [DamnWidget/anaconda_go#3]
One thing I really like in GoSublime is the ability to [explicitly add an import](https://github.com/DisposaBoy/GoSublime/blob/master/USAGE.md#package-imports). It would be great if AnacondaGO supported that feature!

**Comment by DamnWidget** [comment]
Is not `goimports` making that feature irrelevant?

**Comment by alecthomas** [comment]
No, because it doesn't cover the use case of when you want to explore a package. For example, you remember that `encoding/xml` has a function to escape text, but you don't remember the exact name. By explicitly importing the package, you can then type `xml.<tab>` to explore the package. This can not be achieved with `goimports` because it relies on a fully qualified symbol to perform importing. That is, you would already need to know the symbol name ahead of time.

**Comment by DamnWidget** [comment]
For that specific use case you can use the `AnacondaGO: Explore Packages` command then write `xml` and look for the function in the list (that you can fuzzy search as well).

Will that cover the related use cases that this GoSublime feature covers?

**Comment by alecthomas** [comment]
Aaah. I'll take a look.

**Comment by alecthomas** [comment]
As discussed in Gitter, I was unable to test this due to an issue with indexing on OSX.

**Comment by alecthomas** [comment]
More discussion on Gitter ([February 17, 2017 10:18 AM](https://gitter.im/DamnWidget/anaconda?at=58a6335821d548df2c1caedf))

**Comment by DamnWidget** [comment]
I've been thinking about this and I think the best solution here is to make the completer smarter and able to complete things that are not in the buffer.

**Comment by alecthomas** [comment]
That would be ideal in my mind.
